_authors/ctsai.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Cho Tsai
+short_name: ctsai
+twitter: 
```

Comment:
Can you remove this line if you are not using it?

Reply:
Moved the file and removed the line!